test/fuzz/test_runner.py

```diff
@@ -152,6 +169,36 @@ def main():
         )
 
 
+def generate_corpus_seeds(*, fuzz_pool, build_dir, seed_dir, targets):
+    """Generates new corpus seeds.
+
+    Run {targets} without input, and outputs the generated corpus seeds to
+    {seed_dir}.
+    """
+    logging.info("Generating corpus seeds to {}".format(seed_dir))
+
+    def job(command):
+        logging.debug("Running {}..".format(" ".join(command)))
+        logging.debug("Output: {}\n\n".format(
+            subprocess.run(command, check=True, stderr=subprocess.PIPE,
+                           text=True).stderr
```

Comment:
```suggestion
                           universal_newlines=True).stderr
```

otherwise it would be python3.7 only